.gitignore

```diff
@@ -0,0 +1,5 @@
+
+.npmrc
+node_modules
+.idea
+custom-image-export.lke/buildcache
```

Comment:
it's bad practice to push the whole built artifact in the codebase. wheneve you change something, you also have to change the artifact making PR unreadable.
Instead, customers should be able to build very easily with npm run build for example.

(also we are going to ship *.lke packages as releases)